test/e2e/conftest.py

```diff
@@ -13,15 +13,55 @@
 
 import pytest
 import boto3
+import logging
+import time
+from dataclasses import dataclass
 
-from acktest.aws.identity import get_region
-from acktest import k8s
+from acktest.resources import random_suffix_name
+from acktest.aws.identity import get_region, get_account_id
+from acktest import k8s as k8sclient
+from acktest.k8s import resource as k8s
 
+from e2e import CRD_GROUP, CRD_VERSION, load_lambda_resource
+from e2e.service_bootstrap import LAMBDA_FUNCTION_FILE_ZIP
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources    
+
+FunctionPackageTypeImage = "Image"
+FunctionPackageTypeZip = "Zip"
+FunctionTestingRoleBasic = "basic"
+FunctionTestingRoleESM = "esm"
+
+@dataclass
+class ResourceWaitTimes:
+    Create: int
+    Update: int
+    Delete: int
+
+@dataclass
+class ServiceWaitTimes:
+    Function: ResourceWaitTimes
+    Alias: ResourceWaitTimes
+    CodeSigningConfig: ResourceWaitTimes
+    EventSourceMapping: ResourceWaitTimes
+    FunctionURLConfig: ResourceWaitTimes
+
+Wait = ServiceWaitTimes(
+    Function = ResourceWaitTimes(30, 30, 30),
+    Alias = ResourceWaitTimes(10, 10, 10),
+    CodeSigningConfig = ResourceWaitTimes(10, 10, 10),
+    EventSourceMapping = ResourceWaitTimes(20, 20, 20),
+    FunctionURLConfig = ResourceWaitTimes(30, 10, 10),
+)
```

Comment:
I think I'd prefer to see each of these defined in their respective test files. They can be cross-imported if we need to access them in other tests.